module/VuFind/src/VuFind/ILS/Driver/Demo.php

```diff
@@ -1390,6 +1390,38 @@ public function getMyTransactionHistory($patron, $params)
         ];
     }
 
+    /**
+     * Purge Patron Transaction History
+     *
+     * @param array  $patron The patron array from patronLogin
+     * @param ?array $ids    IDs to purge, or null for all
+     *
+     * @throws ILSException
+     * @return array Associative array of the results
+     */
+    public function purgeTransactionHistory(array $patron, ?array $ids): array
+    {
+        $this->checkIntermittentFailure();
```

Comment:
Do we need to add this method to [Failure_Probabilities] in Demo.ini to make this easier to configure?